node/src/GlideClient.ts

```diff
@@ -589,51 +589,48 @@ export class GlideClient extends BaseClient {
      * Returns information about the function that's currently running and information about the
      * available execution engines.
      *
-     * See https://valkey.io/commands/function-stats/ for details.
+     * FUNCTION STATS runs on all nodes of the cluster, including primary and replicas.
+     * The response includes a mapping from node address to the command response for that node.
      *
-     * since Valkey version 7.0.0.
+     * See https://valkey.io/commands/function-stats/ for more details.
      *
-     * @returns A `Record` with two keys:
-     *     - `"running_script"` with information about the running script.
-     *     - `"engines"` with information about available engines and their stats.
+     * Since: Valkey version 7.0.0.
```

Comment:
We have a new format for these sections, please update

Reply:
in case it helps, here's the PR for reference: https://github.com/valkey-io/valkey-glide/pull/2111